.github/workflows/cargo.yml

```diff
@@ -192,17 +192,14 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: "16"
+          cache: "yarn"
 
       # We explicitly do this to cache properly.
       - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
 
-      - uses: actions/setup-node@v2
```

Comment:
this one's dupe, there's setup in https://github.com/swc-project/swc/pull/3015/files#diff-c405e4afd304c7295273c10a937f4a1e3d7853ff71b4066c487e1f3c8291b690L192 already.